otter/models/cass.py

```diff
@@ -1682,6 +1694,54 @@ def health_check(self):
         return d
 
 
+@implementer(IScalingGroupServersCache)
+class CassScalingGroupServersCache(object):
+    """
+    Collection of cache of scaling group servers
+    """
+
+    def __init__(self, tenant_id, group_id):
+        self.tenant_id = tenant_id
+        self.group_id = group_id
+        self.table = "servers_cache"
+        self.params = {"tenant_id": tenant_id, "group_id": group_id}
+
+    @do
+    def get_servers(self):
+        query = ("SELECT server_blob, last_update FROM {cf} "
```

Comment:
Oops, sorry I didn't catch this before.  :(  I think in the schema this is `last_fetch`, not `last_update`